03-Export-Tags-Authors-Stories.py

```diff
@@ -42,18 +42,21 @@ def write_csv(filename, columns):
   if args.archive_type == 'AA':
```

Comment:
This is outside the remit of this PR so we can address this separately, but just noting that this code needs a good refactor because removing the eFiction code has left the "AA" code on its own, looking stupid in `if` statements with no alternative branch! It's all "AA" now...